stdlib/runtime-api/src/main/java/org/ballerinalang/stdlib/runtime/nativeimpl/GetCallStack.java

```diff
@@ -30,6 +39,13 @@
 public class GetCallStack {
 
     public static BArray getCallStack() {
-        return ErrorCreator.createError(StringUtils.fromString("")).getCallStack();
+        List<StackTraceElement> filteredStack = ErrorCreator.createError(StringUtils.fromString("")).getCallStack();
+        Type recordType = ValueCreator.createRecordValue(BLangConstants.BALLERINA_RUNTIME_PKG_ID,
```

Comment:
Shall we define BALLERINA_RUNTIME_PKG_ID inside stdlib. if not everytime we  change runtime package version , we need to build and release lang as well.